logserver.c

```diff
@@ -498,7 +498,7 @@ static int logserver_handle_msg(struct logserver_msg *msg)
 static int logserver_epoll_command(int fd, int cmd)
 {
 	struct epoll_event ev;
-	ev.events = EPOLLIN;
+	ev.events = EPOLLIN | EPOLLRDHUP;
```

Comment:
we dont need EPOLLHUP here?

Reply:
No because EPOLLHUP is always reported by epoll_wait